### PR TITLE
Update versions to match supported versions

### PR DESCRIPTION
**Pull request by aurorasmiles**
Updates the version labeling to match our current supported version list

### CODE REVIEW

File: script/fawe.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ var projects = {
 		],
 		"config_path": "plugins"
 	},
-	"Spigot 1.16.5 - 1.18.2": {
+	"Spigot 1.16.5": {
```
Comment (dordsor21) [inline script/fawe.js:32]: I'm guessing the link for this is probably able to be updated to https://ci.athion.net/job/FastAsyncWorldEdit/588/ as I believe the commit after is what effectively broken 1.16.5 for real

Comment (aurorasmiles) [reply script/fawe.js:32]: done

File: script/fawe.js [hunk 2]
```diff
@@ -29,11 +29,11 @@ var projects = {
...
 		"download": [
 			{
 				"label": "Download FAWE",
-				"url": "https://github.com/IntellectualSites/download/raw/gh-pages/artifacts/Fawe/FastAsyncWorldEdit-Bukkit-2.2.1-SNAPSHOT-193.jar",
+				"url": "https://ci.athion.net/job/FastAsyncWorldEdit/588/",
```
Comment (dordsor21) [inline script/fawe.js]: Ah, might as well link directly to https://ci.athion.net/job/FastAsyncWorldEdit/588/artifact/artifacts/FastAsyncWorldEdit-CLI-2.8.2-SNAPSHOT-588.jar aha

Comment (aurorasmiles) [reply script/fawe.js]: i'm sure people would appreciate the direct link to the CLI :p

**Review by dordsor21 — APPROVED** [review]

**Pull request merged**